### PR TITLE
Changelog for 1.7.8.0-RC-1

### PR DESCRIPTION
**Pull request by sowbiba**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Changelog for 1.7.8.0 RC 1
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| How to test?      | -
| Possible impacts? | -

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25123)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]
Nothing new in contributors?

**Review by matks — DISMISSED** [review]

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -685,6 +686,7 @@ GitHub contributors:
 - sadlyblue
 - sagaradonis
 - sallemiines
+- Sam
```
Comment (matks) [inline CONTRIBUTORS.md:689]: Is it juraj1000 ?

Comment (sowbiba) [reply CONTRIBUTORS.md:689]: Nothing indicates that it's the same person

Comment (sowbiba) [reply CONTRIBUTORS.md:689]: You'll see his mail in the Changelog project PR

**Review by jolelievre — DISMISSED** [review]

**Review by kpodemski — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Pull request merged**